pkg/wpa/passphrase/passphrase.go

```diff
@@ -0,0 +1,37 @@
+package passphrase
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"fmt"
+
+	"golang.org/x/crypto/pbkdf2"
+)
+
+const ResultFormat = `network={
+	ssid="%s"
+	#psk="%s"
+	psk=%s
+}
+`
+
+func errorCheck(essid string, pass string) error {
+	if len(pass) < 8 || len(pass) > 63 {
```

Comment:
where do 8 and 63 come from? They need to be constants somewhere?

Reply:
I got it from the source code. The original code just put 8 and 63 in there.
http://docs.ros.org/diamondback/api/wpa_supplicant/html/wpa__passphrase_8c_source.html

Reply:
ok, that's fine then.

Reply:
let's declare them as constants anyway. MinPasswordLen and MaxPasswordLen or something.

Reply:
Fixed